specification/peering/resource-manager/Microsoft.Peering/preview/2019-03-01-preview/peering.json

```diff
@@ -0,0 +1,1449 @@
+          "description": "The contact information of the peer."
+        },
+        "peerName": {
+          "description": "The name of the peer.",
+          "type": "string"
+        },
+        "validationState": {
+          "description": "The validation state of the ASN associated with the peer.",
+          "enum": [
+            "None",
+            "Pending",
+            "Approved",
+            "Failed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "validationState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ContactInfo": {
+      "description": "The contact information of the peer.",
+      "type": "object",
+      "properties": {
+        "emails": {
+          "description": "The list of email addresses.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "phone": {
+          "description": "The list of contact numbers.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PeerAsnListResult": {
+      "description": "The paginated list of [T].",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of [T].",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PeerAsn"
+          }
+        },
+        "nextLink": {
+          "description": "The link to fetch the next page of [T].",
+          "type": "string"
+        }
+      }
+    },
+    "PeeringLocationListResult": {
+      "description": "The paginated list of peering locations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of peering locations.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PeeringLocation"
+          }
+        },
+        "nextLink": {
+          "description": "The link to fetch the next page of peering locations.",
+          "type": "string"
+        }
+      }
+    },
+    "PeeringLocation": {
+      "description": "Peering location is where connectivity could be established to the Microsoft Cloud Edge.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "The kind of peering that the peering location supports.",
+          "enum": [
+            "Direct",
+            "Exchange"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "kind",
+            "modelAsString": true
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/PeeringLocationProperties",
+          "description": "The properties that define a peering location.",
+          "x-ms-client-flatten": true
+        },
+        "name": {
+          "description": "The name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
```

Comment:
is this also a "resource" shoud it share the "resource" definition with "Peer" and should it be tagged x-ms-azurre-resouce?

Reply:
no it is a proxy read only resource.